packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -74,6 +74,18 @@ endpoint_tests:
     type:
       - master
   - tests:
+      # This test checks whether AR's `/service` endpoint is able to route to
+      # the correct service even though the request url contains duplicate
+      # slashes.  Duplicate slashes is AFAIK not a valid RFC behaviour, but
```

Comment:
Also "what is 'the correct service'"? is a question that comes to mind. Is it possible that someone would want a different endpoint at `baz//bat` than `baz/bat`?